frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -249,6 +250,42 @@ export const createRoute = (
   return k8sCreate(RouteModel, route, dryRun ? dryRunOpt : {});
 };
 
+export const createResourceLimit = (
+  formData: GitImportFormData,
+  imageStream: K8sResourceKind,
+  dryRun: boolean,
+): Promise<K8sResourceKind> => {
+  const {
+    name,
+    project: { name: namespace },
+    application: { name: application },
+    labels: userLabels,
+    limits: resource,
+  } = formData;
+
+  const defaultLabels = getAppLabels(name, application, imageStream.metadata.name);
+
+  const resourceLimit = {
+    kind: 'ResourceQuota',
```

Comment:
Why are we creating a quota? If you're setting limits on a pod, the limits are set in the workload pod template. Quota is the entire quota for the project and can only be created admins.

Reply:
@spadgett Which API call should be to call for the Resource Limit creation? and where I can see it in the console if it is successfully created or not?